Gemfile

```diff
@@ -28,10 +28,7 @@ gem 'valid_email'
 gem 'rack-attack'
 gem 'readthis'
 gem 'rqrcode'
-
-# unreleased feature via: https://github.com/onelogin/ruby-saml/pull/345
-gem 'ruby-saml', github: 'onelogin/ruby-saml', branch: 'master'
-
+gem 'ruby-saml'
```

Comment:
I meant lock it down so that future `bundle update`s or `bundle update ruby-saml` do not upgrade to the next major release and possibly break functionality. 
```
gem 'ruby-saml', '~> 1.4.0'
```